recipes/guidata/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "guidata" %}
+{% set version = "1.7.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://github.com/PierreRaybaut/{{ name }}.git
```

Comment:
Can you use a tarball instead? We discourage the use of `git_url` to avoid cloning the repo.